docker/nginx/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.19.0-alpine
+FROM nginx:1.23.4-alpine
```

Comment:
Shouldn't this be placed in a separate PR?

Reply:
You're right. I created a separate MR: https://github.com/dflss/boards-of-django/pull/52